docker/Dockerfile.devito

```diff
@@ -18,6 +23,7 @@ RUN python3 -m venv /venv && \
     /venv/bin/pip install --no-cache-dir --upgrade pip && \
     /venv/bin/pip install --no-cache-dir jupyter && \
     /venv/bin/pip install --no-cache-dir wheel && \
+    eval "$MPI4PY_FLAGS /venv/bin/pip install --no-cache-dir mpi4py" && \
```

Comment:
Where does MPI4PY_FLAGS come from? A quick google indicates zero hits. Add link/comment?

Reply:
It's just a made up one I made to handle all our different compilers and arches